### PR TITLE
fix(ssh_tunnel): Address Base PR comments from peer review

### PR DESCRIPTION
**Pull request by Antonio-RiveroMartnez**
### SUMMARY
Address code reviews comments:

- https://github.com/apache/superset/pull/21912#discussion_r1026958824
- https://github.com/apache/superset/pull/21912#discussion_r1035415356 
- https://github.com/apache/superset/pull/21912#discussion_r1035558825
- https://github.com/apache/superset/pull/21912/files#r1028278374
- https://github.com/apache/superset/pull/21912/files#r1028286296
- https://github.com/apache/superset/pull/21912/files#r1026937885
- https://github.com/apache/superset/pull/21912/files#r1026956904
- https://github.com/apache/superset/pull/21912/files#r1035561067
- https://github.com/apache/superset/pull/21912/files#r1035558379
- https://github.com/apache/superset/pull/21912/files#r1035558237

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/databases/ssh_tunnel/models.py [hunk 1]
```diff
@@ -41,15 +42,17 @@ class SSHTunnel(Model, AuditMixinNullable, ExtraJSONMixin, ImportExportMixin):
...
+    )
     database: Database = relationship(
         "Database",
         backref=backref("ssh_tunnels", cascade="all, delete-orphan"),
         foreign_keys=[database_id],
     )
 
-    server_address = sa.Column(EncryptedType(sa.String, app_config["SECRET_KEY"]))
-    server_port = sa.Column(EncryptedType(sa.Integer, app_config["SECRET_KEY"]))
+    server_address = sa.Column(sa.Text)
```
Comment (eschutho) [inline superset/databases/ssh_tunnel/models.py:54]: why aren't these encrypted any more?

Comment (Antonio-RiveroMartnez) [reply superset/databases/ssh_tunnel/models.py:54]: I was addressing this comment: https://github.com/apache/superset/pull/21912/files#r1035561067 which mentions we don't need those fields encrypted. Do we want to encrypt them? if yes I can rollback.

Comment (craig-rueda) [reply superset/databases/ssh_tunnel/models.py:54]: They shouldn't need to be encrypted, IMO. Server hostname/port shouldn't contain sensitive info :)

Comment (eschutho) [reply superset/databases/ssh_tunnel/models.py:54]: looks good.

**Review by eschutho — APPROVED** [review]
thanks @Antonio-RiveroMartnez. Looks great!

**Comment by rusackas** [comment]
Re-running CI... will merge if it passes 🤞

**Pull request merged**